rocketpool-cli/minipool/close.go

```diff
@@ -72,6 +73,14 @@ func closeMinipools(c *cli.Context) error {
 				unwithdrawnMinipools = append(unwithdrawnMinipools, mp)
 			}
 		}
+		if eth.WeiToEth(mp.DepositBalance) != 16 {
```

Comment:
we should move this to L61 so we can refactor the `if mp.CanClose` case to contain a `continue`...

Reply:
I don't see how this would change an specific minipool CanClose info.

Reply:
it would just make the code more future proof